jwst/cube_build/blot_cube_build.py

```diff
@@ -281,19 +231,135 @@ def blot_overlap_quick(self, model, xcenter, ycenter, xstart,
             if len(index_x[0]) > 0 and len(index_y[0]) > 0:
                 d1pix = np.array(x_cube[ipt] - xcenter[index_x])
                 d2pix = np.array(y_cube[ipt] - ycenter[index_y])
-                dxy=[]
-                for dy in d2pix:
-                    for dx in d1pix:
-                        dxy.append((dx * dx) + (dy * dy))
+
+                dxy = [ (dx*dx + dy*dy)  for dy in d2pix for dx in d1pix]
                 dxy = np.sqrt(dxy)
                 weight_distance = np.exp(-dxy)
                 weighted_flux = weight_distance * flux_cube[ipt]
                 index2d = [iy * blot_xsize + ix for iy in index_y[0] for ix in (index_x[0]+xstart)]
                 blot_flux[index2d] = blot_flux[index2d] + weighted_flux
                 blot_weight[index2d] = blot_weight[index2d] + weight_distance
 
+        t1 = time.time()
+        log.debug("Time to blot median image to input model = %.1f s" % (t1-t0,))
```

Comment:
Not a big deal, but could change this to use the new python string formatting syntax:
```
log.debug(f"Time to blot median image to input model = {t1-t0:.1f}")
```

Reply:
fixed